### ISSUE TITLE
cannot install search plugins

### ISSUE THREAD

**Issue by DirtyMarx** [qbittorrent/search-plugins#219]
I just did a fresh install and cannot install any plugins
![qbittorrent_CvijD5JCjM](https://user-images.githubusercontent.com/101524909/206091337-8dcd8d54-a14e-4800-998e-38138f36f413.png)

**Comment by Melodey757** [comment]
I'm having the same problem... Search Plugins were there yesterday, gone today.. re-installing didn't work..

**Comment by sausje** [comment]
Yep, updated and wanted to try out search for the first time... Welp, it doesn't work, and it seems for a month already... Bad...

**Comment by p060477** [comment]
me too....this is a very big BUG and the silence from dev is so astonishing...what a shame... :(

**Comment by Protey** [comment]
I confirm the problem is still not solved!!!
